### ISSUE TITLE
forwardattachment patch for RC 0.4-stable

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#95]
```
Attached a patch for RC 0.4-stable to fix the forwardattachment plugin.
```

Original issue reported on code.google.com by `iez...@gmail.com` on 13 Aug 2010 at 5:27

Attachments:
- [forwardattachment-0.4.patch](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-95/comment-0/forwardattachment-0.4.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks!
```

Original comment by `rol...@liebl.ath.cx` on 13 Aug 2010 at 5:31
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3878.1.
```

Original comment by `rol...@liebl.ath.cx` on 14 Aug 2010 at 4:47
- Changed state: **Fixed**
